src/server.js

```diff
@@ -26,7 +26,7 @@ const startServer = async (portNumber) => {
         maxAge: 1000*60*5
       }
     }))
-
+    app.patch('/api/users/:id',updateDBPassword)
```

Comment:
could you move this to the other `/api/users` routes so the bunch of them are close to each other?